### PR TITLE
Add close button to license dialog

### PR DESCRIPTION
**Pull request by peterNordin**

### CODE REVIEW

File: HopsanGUI/Dialogs/LicenseDialog.cpp [hunk 1]
```diff
@@ -44,7 +44,6 @@ LicenseDialog::LicenseDialog(QWidget *pParent) :
     resize(640,480);
     setWindowTitle(tr("Hopsan License"));
     setAttribute(Qt::WA_DeleteOnClose, true);
-    setPalette(QPalette(QColor("gray"), QColor("whitesmoke")));
```
Comment (robbr48) [inline HopsanGUI/Dialogs/LicenseDialog.cpp]: Should this line really be removed?

**Comment by peterNordin** [comment]
Yes, I should have made a separate commit. It was required before for some reason to make the license text visible. But I have rewritten the dialog now and it looks strange if I force that style. Now it looks OK with both native and custom hopsan style

File: HopsanGUI/Dialogs/LicenseDialog.cpp (hunk 1 above)
Comment (peterNordin) [reply HopsanGUI/Dialogs/LicenseDialog.cpp]: Yes, I should have made a separate commit. It was required before for some reason to make the license text visible. But I have rewritten the dialog now and it looks strange if I force that style. Now it looks OK with both native and custom hopsan style

Comment (peterNordin) [reply HopsanGUI/Dialogs/LicenseDialog.cpp]: I can rebase, lets try that before merge :)

**Pull request merged**